### ISSUE TITLE
Set Full Calendar weekly view date to ISO format

### ISSUE THREAD

**Issue by tingmelvin** [davish/obsidian-full-calendar#52]
Probably on your to-do list. Currently it's in an American-style format (Mar 6 - 12, 2022), it'll be lovely if it could be turned into YYYY-MM-DD or something that the user can define!

Thanks for the work on the plugin!

**Comment by Quisquose** [comment]
Agreed. I came here to say the same thing. There is currently no standard date format available (only USA-specific date format) which means that for everyone else the day and month seem backwards. This is particularly confusing in places where only numbers are shown. If I see 04/06, to me that's that's 4th June not 6th April.

Hopefully config settings will be provided soon, so that users can set the date to their liking to suit their locale. Getting the date correct is obviously a pretty high priority for any calendar app.

**Comment by davish** [comment]
@Quisquose which places in the app only show numbers? The one I can think of is the datepicker for events in the modal. In that case, I don't think the HTML default datepicker supports any customization, and so part of this task will be investigating a custom datepicker.

**Comment by davish** [comment]
Ah, in week view there's `Sat 5/7`, for example instead of `7 May`. As an American, I'm not as familiar with other localized date formats, I'll look for a way to let people configure this in JSON.

Relevant documentation:
https://fullcalendar.io/docs/day-header-render-hooks
https://fullcalendar.io/docs/slotLabelFormat
https://fullcalendar.io/docs/date-formatting

**Comment by kevfrost** [comment]
Came here to agree with my peers above.

**Comment by Alejandro-Stolz** [comment]
> Ah, in week view there's `Sat 5/7`, for example instead of `7 May`. As an American, I'm not as familiar with other localized date formats, I'll look for a way to let people configure this in JSON.
> 
> Relevant documentation: https://fullcalendar.io/docs/day-header-render-hooks https://fullcalendar.io/docs/slotLabelFormat https://fullcalendar.io/docs/date-formatting

Is there already a way to configure this maybe in the data.json file or any advancements? Thank you

**Comment by tophee** [comment]
> Is there already a way to configure this maybe in the data.json file or any advancements?

@Alejandro-Stolz Did you find a place where the date format can be changed?

**Comment by Alejandro-Stolz** [comment]
> > Is there already a way to configure this maybe in the data.json file or any advancements?
> 
> @Alejandro-Stolz Did you find a place where the date format can be changed?

Sadly not. I've been using the calendar non-stop since I opened the issue and I've gotten used to it.
